index.html

```diff
@@ -2,15 +2,13 @@
 <html>
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-  <script src="bower_components/webcomponentsjs/webcomponents.js"></script>
-  <link rel="import" href="bower_components/polymer/polymer.html">
-  <link rel="import" href="bower_components/core-component-page/core-component-page.html">
-</head>
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../iron-component-page/iron-component-page.html">
```

Comment:
this is top-level so the relative paths point higher than the web root. Am I just forgetting how this is supposed to work? Sorry, it's been a while since I looked at this stuff. :flushed: